patches/minecraft/net/minecraft/world/entity/npc/CatSpawner.java.patch

```diff
@@ -1,11 +1,10 @@
 --- a/net/minecraft/world/entity/npc/CatSpawner.java
 +++ b/net/minecraft/world/entity/npc/CatSpawner.java
-@@ -86,8 +_,9 @@
+@@ -86,8 +_,8 @@
        if (cat == null) {
           return 0;
        } else {
 +         cat.m_20035_(p_35334_, 0.0F, 0.0F); // Fix MC-147659: Some witch huts spawn the incorrect cat
```

Comment:
Re: this discussion in discord.
https://discord.com/channels/313125603924639766/852298000042164244/1006758427491586209

We are not sure that this comment is valid anymore